src/xeus_sqlite_interpreter.cpp

```diff
@@ -140,9 +135,7 @@ void interpreter::create_db(const std::vector<std::string> tokenized_code)
     }
     catch(const std::runtime_error& err)
     {
-        m_traceback.push_back("Error: " + (std::string)err.what());
-        publish_execution_error("Error", err.what(), m_traceback);
-        m_traceback.clear();
+        throw std::runtime_error("Error: " + (std::string)err.what());
```

Comment:
Same here :)

Reply:
I see, thanks for the input!

Reply:
But here, I'm actually using the `std::runtime_error` error. Shouldn't I also use a `catch` here?

Reply:
Not sure I get what you are doing with the `m_traceback` actually?

Reply:
I think I removed it every where now, I was using it wrong. Can change it back to normal.

Reply:
Ok, so maybe this code can be removed too and simplified :)

Reply:
I'm not sure if I should create an `if` here, it's being useful. I don't really get what you mean @martinRenou, please let me know if you'd like me to actually reformulate it to use an `if`.

Reply:
I did not tell you to use an `if`. I was saying that this code can be removed.

Your code can be simplified from something like:
```cpp
try
{
  // Actual code
}
catch(exception)
{
   throw exception;
}
```

To:

```cpp
// Actual code
```

You are basically re-throwing the same exception with slightly the same error message. Just let the exception propagate? It will be catched later by your code.